pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/JSONSchemaHandler.java

```diff
@@ -18,26 +18,31 @@
  */
 package org.apache.pulsar.sql.presto;
 
-import com.dslplatform.json.DslJson;
 import com.facebook.presto.spi.type.Type;
-
+import com.google.common.annotations.VisibleForTesting;
 import io.airlift.log.Logger;
 
-import java.io.IOException;
 import java.math.BigDecimal;
 import java.util.List;
-import java.util.Map;
 
 import io.netty.buffer.ByteBuf;
 import io.netty.util.concurrent.FastThreadLocal;
+import org.apache.pulsar.client.api.PulsarClientException;
+import org.apache.pulsar.client.impl.schema.generic.GenericJsonRecord;
+import org.apache.pulsar.client.impl.schema.generic.GenericJsonSchema;
+import org.apache.pulsar.common.api.raw.RawMessage;
+import org.apache.pulsar.common.naming.TopicName;
+import org.apache.pulsar.common.schema.SchemaInfo;
 
 public class JSONSchemaHandler implements SchemaHandler {
 
     private static final Logger log = Logger.get(JSONSchemaHandler.class);
 
     private List<PulsarColumnHandle> columnHandles;
 
-    private final DslJson<Object> dslJson = new DslJson<>();
```

Comment:
Why are we changing this?  DSL Json library is much faster than any other json deserialization library.